react/features/chat/components/native/ChatButton.ts

```diff
@@ -72,7 +73,7 @@ function _mapStateToProps(state: IReduxState, ownProps: any) {
         _isPollsDisabled: arePollsDisabled(state),
 
         // The toggled icon should also be available for new polls
-        _unreadMessageCount: getUnreadCount(state) || getUnreadPollCount(state),
+        _unreadMessageCount: getUnreadCount(state) || getUnreadPollCount(state) || getUnreadFilesCount(state),
```

Comment:
Can you update the comment, or remove it altogether, since it adds nothing?

Reply:
sure..